src/index.tsx

```diff
@@ -118,6 +118,8 @@ function hitTest(element: ExcalidrawElement, x: number, y: number): boolean {
     const y2 = getElementAbsoluteY2(element);
 
     return x >= x1 && x <= x2 && y >= y1 && y <= y2;
+  } else if (element.type === "selection") {
+    return false;
```

Comment:
Without this, we're throwing the error below on clicks on the canvas. This was reproducible in the production too with a combination of right-clicks and left-clicks.
I'm not really sure if that's the best way to handle it

Reply:
Could you add a console.warn here, so that at some point we figure out why there's a race condition and fix it?

Reply:
I'm not really good with warning messages, what do you suggest? 😅

Reply:
"This should not happen, we need to investigate why it does."

Reply:
Fixed https://github.com/excalidraw/excalidraw/pull/85/commits/94ed180584ba01cdbf995350e83a9ddbbc8d1675